README.md

```diff
@@ -153,6 +155,24 @@ The table below shows all the skills implemented in the bot:
   
 </table>
 
+<table border="1">
+   <tr>
+   <th>Skill Group</th>
+   <th>Description</th>
+   <th>Link to loading script</th>
+   </tr>
+   
+   <!-- Enterprise Search -->
+   
+   <tr>
+      <td rowspan="5">Enterprise Search</td>
+      <td>Q&A based on SQUAD Dataset</td>
+      <td>Load and search the https://huggingface.co/datasets/rajpurkar/squad dataset.</td>
```

Comment:
Do you think this dataset is appropriate for a demo bot? I think a different FAQ dataset like https://huggingface.co/datasets/CShorten/CDC-COVID-FAQ would be more appropriate.

Squad has quite strange QnA like "Which prize did Frederick Buechner create?" with mostly single word answers.
COVID FAQ has much shorter questions and the answers are better phrased. I can make the changes regarding the dataset in the script in this PR if you want

Reply:
Sure, if you think it would showcase it better. But it is more about "how" here than "what", right, so I would be fine with this one too